modules/installation-configuration-parameters.adoc

```diff
@@ -596,7 +600,8 @@ accounts for the dramatically decreased machine performance.
 |The only supported value is `3`, which is the default value.
 
 |`credentialsMode`
```

Comment:
Table 4 seems to have fixed width columns, causing it to hide the text of the columns in the preview. Ideally, all of the text should appear without the need to horizontally scroll the table, which might not be obvious to readers.

Reply:
I believe this is due to the parameters in table 4 that begin with `platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey`. They were added in another 4.12 ticket and since they're all technically one word, the table doesn't wrap them. I don't know of a better way to handle these deeply nested parameters. Do you?

Reply:
I really don't. You have to scroll down to the bottom of the table to get the horizontal navigation. It looks like it was put in so that the parameters don't wrap, which forces the description column under the nav and forces you to scroll. I don't know that it would be better with parameters wrapping.

Reply:
Ah, I see your point. This hasn't been an issue for me because I use a laptop and I can use the trackpad to easily side-scroll without having to use the horizontal bar at the bottom of the table. I will bring this up at our next docs team meeting and see if there are any ideas on how to work around this.